docs/persistence.rst

```diff
@@ -0,0 +1,81 @@
+.. _persistence:
+
+Secure persistence with skops
+=============================
+
+.. warning::
+
+   This feature is very early in development, which means the API is
+   unstable and it is **not secure** at the moment. Therefore, use the same
+   caution as you would for ``pickle``: Don't load from sources that you
+   don't trust. In the future, more security will be added.
+
+Skops offers a way to save and load sklearn models without using :mod:`pickle`.
+The ``pickle`` module is not secure, but with skops, you can securely save and
+load sklearn models without using ``pickle``.
+
+Pickle is the standard serialization format for sklearn and for Python in
+general. One of its biggest advantages is it can be used for almost all Python
```

Comment:
The sentence is hard for me to parse, maybe:

```suggestion
general. One of the main advantages of ``pickle`` is that it can be used for almost all Python
```